pyanaconda/core/dbus.py

```diff
@@ -96,18 +96,26 @@ def __init__(self, default_name):
 
     def match_type(self, exception_type):
         """Match every Python exception raised on the server side."""
+        del exception_type
```

Comment:
Seems to me that this is not a correct python pattern. Based on my knowledge and search the correct pattern is `_` prefix. As discussed in comments to this PEP https://discuss.python.org/t/pep-640-unused-variable-syntax/5490/2 (which was rejected because it wanted to use `unused` keyword). However, that is changing the function signature :(. 

I'm not strictly against it but I don't remember I would see pattern like this elsewhere.

Reply:
@jkonecny12 @M4rtinK It's in fact recommended by pylint: 
https://pycodequ.al/docs/pylint-messages/w0613-unused-argument.html#how-to-fix

@M4rtinK take care that you cant always use underscore. One can use for positional arguments but not for keyword arguments. https://stackoverflow.com/a/14836005

In the case, when we are overriding functions, I am not sure change the function signature, let's check that.

Reply:
@jkonecny12 not sure if you saw, but this is already obsolete, and adjusted to not use the del pattern. Please re-review.